data-files/samples/trials.Experiment.Any

```diff
@@ -0,0 +1,142 @@
+{ 
+    description = "trials"; 
+    frameRate = 120; 
+    showHUD = true;
+    renderWeaponStatus = true;
+    cooldownMode = "ring";
+    cooldownColor = Color4(0.7,1.0,0.9,0.75);
+            
+    weapon = { 
+        id = "2hit"; 
+        firePeriod = 0.3;
+        damagePerSecond = 2;
+        autoFire = false;
+    }; 
+    sessions = ( 
+        { 
+            // This session tests out different weapon cooldowns and cooldown indicators while maintaining 2 hits to kill
+            id = "1target_cd"; 
+            description = "1 target cooldown"; 
+            trials = ( 
+                { 
+                    count = 2; 
+                    ids = ( "static"); 
+                }, 
+                { 
+                    count = 2; 
+                    ids = ( "static"); 
+                    cooldownMode = "box";
+                }, 
+                { 
+                    count = 2; 
+                    ids = ( "static"); 
+                    cooldownMode = "box";
+                    weapon = { 
+                        id = "2hit"; 
+                        firePeriod = 0.5;
+                        damagePerSecond = 1.1;
+                        autoFire = false;
+                    }; 
+                }, 
+                { 
```

Comment:
This trial looks identical to the one above it, am I missing something?

Reply:
yep, meant to leave out the cooldown type to fall back to the experiment default.